frontend/src/ManageOrgUsers/ManageOrgUsers.jsx

```diff
@@ -54,20 +57,34 @@ class ManageOrgUsers extends React.Component {
         toast.success('User role has been updated', { hideProgressBar: true, position: 'top-center' });
         this.setState({ idChangingRole: null });
       })
-      .catch(( { error } ) => {
+      .catch(({ error }) => {
         toast.error(error, { hideProgressBar: true, position: 'top-center' });
         this.setState({ idChangingRole: null });
       });
   };
 
+  archiveOrgUser = (id) => {
+    this.setState({ archivingUser: id });
+
+    organizationUserService
+      .archive(id)
+      .then(() => {
+        toast.success('User has been removed', { hideProgressBar: true, position: 'top-center' });
+        this.setState({ archivingUser: null });
+        this.fetchUsers();
```

Comment:
I am not as comfortable with React as I am with Ruby. It seems a little ugly to re-fetch the list after archiving the user. However, I am struggling to figure out how to do it. This is getting the job done.

Reply:
You are right. There are a few ways to do this without an additional query. Let's create an issue for this and prioritise later?

Reply:
Absolutely. Will make an issue as soon as this is merged.